src/algebra/continued_fractions/computation/approximations.lean

```diff
@@ -563,8 +563,7 @@ begin
     suffices : (ifp_succ_n.b : K) * conts.b ≤ ifp_n.fr⁻¹ * conts.b, by rwa [←ifp_succ_n_b_eq_gp_b],
     have : (ifp_succ_n.b : K) ≤ ifp_n.fr⁻¹, from
       int_fract_pair.succ_nth_stream_b_le_nth_stream_fr_inv stream_nth_eq succ_nth_stream_eq,
-    have : 0 ≤ conts.b, from le_of_lt zero_lt_conts_b,
-    mono }
+    exact mul_le_mul_of_nonneg_right ‹_› zero_lt_conts_b.le }
```

Comment:
This change is somewhat worrying - why can't `mono` clear up this goal any more?

Reply:
I found out yesterday that I forgot to tag two lemmas with `mono` so I think this should be fixable back now.

Reply:
Indeed